content/en/docs/setup/install/istioctl/index.md

```diff
@@ -303,344 +303,7 @@ Then run the following `verify-install` command to see if the installation was s
-
-The following example overlay file (`patch.yaml`) demonstrates the type of output manifest patching that can be done:
-
-{{< text yaml >}}
-apiVersion: install.istio.io/v1alpha1
-kind: IstioOperator
-spec:
-  profile: empty
-  hub: docker.io/istio
-  tag: 1.1.6
-  components:
-    pilot:
-      enabled: true
-      namespace: istio-control
-      k8s:
-        overlays:
-          - kind: Deployment
-            name: istiod
-            patches:
-              # Select list item by value
-              - path: spec.template.spec.containers.[name:discovery].args.[30m]
-                value: "60m" # overridden from 30m
-              # Select list item by key:value
-              - path: spec.template.spec.containers.[name:discovery].ports.[containerPort:8080].containerPort
-                value: 1234
-              # Override with object (note | on value: first line)
-              - path: spec.template.spec.containers.[name:discovery].env.[name:POD_NAMESPACE].valueFrom
-                value: |
-                  fieldRef:
-                    apiVersion: v2
-                    fieldPath: metadata.myPath
-              # Deletion of list item
-              - path: spec.template.spec.containers.[name:discovery].env.[name:REVISION]
-              # Deletion of map item
-              - path: spec.template.spec.containers.[name:discovery].securityContext
-          - kind: Service
-            name: istiod
-            patches:
-              - path: spec.ports.[name:https-dns].port
-                value: 11111 # OVERRIDDEN
-{{< /text >}}
-
-Passing the file to `istioctl manifest generate -f patch.yaml` applies the above patches to the default profile output
-manifest. The two patched resources will be modified as shown below (some parts of the resources are omitted for
-brevity):
-
-{{< text yaml >}}
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: istiod
-spec:
-  template:
-    spec:
-      containers:
-      - args:
-        - 60m
-        env:
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              apiVersion: v2
-              fieldPath: metadata.myPath
-        name: discovery
-        ports:
-        - containerPort: 1234
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: istiod
-spec:
-  ports:
-  - name: https-dns
-    port: 11111
----
-{{< /text >}}
-
-Note that the patches are applied in the given order. Each patch is applied over the output from the
-previous patch. Paths in patches that don't exist in the output manifest will be created.
-
-### List item path selection
-
-Both the `istioctl --set` flag and the `k8s.overlays` field in `IstioOperator` CR support list item selection by `[index]`, `[value]` or by `[key:value]`.
-The --set flag also creates any intermediate nodes in the path that are missing in the resource.
+For advance customization of installation configuration please refer [more guides](/docs/setup/additional-setup/customize-installation/) section.
```

Comment:
See [Customizing the installation configuration](/docs/setup/additional-setup/customize-installation/) for additional information on customizing the install.